### PR TITLE
Styled the NavBar

### PR DESCRIPTION
**Pull request by BenLardie**
### What this PR does (required):
- navbar was a skeleton now it has style.
- still will need functionality when messages are ready but the notification indicator is set up

### Screenshots / Videos (front-end only):
Screen Shot 2021-11-02 at 1 33 58 PM

### Any information needed to test this feature (required):
- its a visual test

### CODE REVIEW

File: client/src/components/AuthMenu/AuthMenu.tsx [hunk 1]
```diff
@@ -35,13 +36,27 @@ const AuthMenu = (): JSX.Element => {
   const classes = useStyles();
 
   return (
-    <div>
-      <IconButton aria-label="show auth menu" aria-controls="auth-menu" aria-haspopup="true" onClick={handleClick}>
-        {!!loggedInUser && <AvatarDisplay loggedIn user={loggedInUser} />}
+    <div className={classes.navBar}>
```
Comment (ezeilo-su) [inline client/src/components/AuthMenu/AuthMenu.tsx:39]: Please use MUI `Box` component for HTML `div` element

File: client/src/components/AuthMenu/AuthMenu.tsx [hunk 2]
```diff
@@ -35,13 +36,27 @@ const AuthMenu = (): JSX.Element => {
...
+      <IconButton
+        aria-label="show auth menu"
+        aria-controls="auth-menu"
+        aria-haspopup="true"
+        className={classes.navAvatar}
+        onClick={handleClick}
+      >
+        {!!loggedInUser && (
```
Comment (ezeilo-su) [inline client/src/components/AuthMenu/AuthMenu.tsx:47]: Good to make it boolean :+1:

**Review by ezeilo-su — CHANGES_REQUESTED** [review]
Good work with this :+1: 
#### A few things to always remember when creating PRs:
- Fill the PR template properly
- Link the issue you're addressing to your PR. Check out this [guide](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue) to see how to.

Happy Coding!

**Comment by BenLardie** [comment]
> Good work with this 👍
> 
> #### A few things to always remember when creating PRs:
> * Fill the PR template properly
> * Link the issue you're addressing to your PR. Check out this [guide](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue) to see how to.
> 
> Happy Coding!

You are so quick! I knew I forgot to link issue so I went back to do it but you beat me.

**Review by bonnieli — APPROVED** [review]

**Pull request merged**